eth/backend.go

```diff
@@ -610,24 +611,36 @@ func (s *Ethereum) TxPool() *core.TxPool        { return s.txPool }
 func (s *Ethereum) ChainKV() ethdb.RwKV         { return s.chainKV }
 func (s *Ethereum) NetVersion() (uint64, error) { return s.networkID, nil }
 func (s *Ethereum) NetPeerCount() (uint64, error) {
-	// TODO (mempirate): Get peers for all sentries (internal and external)
-	// and return unique count
-	peers := make(map[string]interface{})
-	for _, ss := range s.sentryServers {
-		ss.Peers.Range(func(key, value interface{}) bool {
-			peerID := key.(string)
-			x, _ := ss.Peers.Load(peerID)
-			peerInfo, _ := x.(*download.PeerInfo)
-			if peerInfo == nil {
+	var sentryPc uint64 = 0
+	if len(s.sentryServers) > 0 {
+		peers := make(map[string]interface{})
```

Comment:
Here you can just call `PeerCount` on the server

Reply:
Right, changed it to use `SentryClient` for both internal and external sentries